src/styles/blocks/header.scss

```diff
@@ -0,0 +1,55 @@
+.header {
+  box-shadow: 0 2px 4px rgba(0, 0, 0, 0.05);
+  background: #fff;
+  justify-content: space-between;
+  align-items: center;
+  padding: 0 50px;
+  display: flex;
+}
+
+.nav__list {
+  list-style: none;
+  font-size: 12px;
+  line-height: 15px;
+  text-transform: uppercase;
+  list-style-type: none;
+  justify-content: end;
+  display: flex;
+}
+
+.nav__item:not(:last-child) {
+  margin-right: 20px;
+}
+
+.nav__link {
+  height: 60px;
+  display: block;
+  padding: 23px 0;
+  text-decoration: none;
+  color: $main-color;
+
+  &:hover {
+    color: $button-color;
+  }
+}
```

Comment:
You should create separate file nav.scss, the same for each your BEM-Block. 

In addition, you shoud use SASS nesting. For exaple
```
.nav {
 &__list {
 }
 
 &__link {
 }
```
 etc.

Reply:
The same thing with pseudoclasses and elements